src/ui/dictionary/dictionaryView.svelte

```diff
@@ -11,11 +11,17 @@
   let query: string = "";
   let promise: Promise<DictionaryWord>;
 
-  function handleBlur() {
-    if (query.trim() != "") {
+  function search() {
+    if (query.trim() !== "") {
       promise = manager.requestDefinitions(query);
     }
   }
+
+  function handleKeyDown(e: KeyboardEvent) {
+    if (e.key === 'Enter') {
```

Comment:
Allows searching by pressing enter. Let me know if you still want to keep the blur handler!

Reply:
I think this is way better, I think the Blur Handler isn't necessary anymore.